docs/course-best-practices.md

```diff
@@ -0,0 +1,519 @@
+# Course management best practices
```

Comment:
Maybe "Advanced course development"? Or something with "Advanced" in the title?

Reply:
Advanced course content development?

Reply:
I might go with something like "Course code management", since adding the word "advanced" makes it seem like this is exclusively for courses that are already large. These changes are actually best to make _before_ writing a ton of course code.